packages/theme/src/themes.ts

```diff
@@ -132,3 +133,8 @@ export const darkTheme = createTheme({
     },
   },
 });
+
+export const customPageTheme = createPageTheme({
```

Comment:
There's no way for users of backstage to switch out this theme, it's essentially now hardcoded here instead of in `@backstage/core`

Reply:
I think I have misunderstood something.